File: fizzbuzz.pas
Program FizzBuzz(output);
var
  number : integer;
  finish : integer;

begin
  number := 0;
  finish := 20;

  while number < finish do
  begin
      number := number + 1;

      if number mod 15 = 0 then
        writeln('FizzBuzz!')
      else if number mod 3 = 0 then
        writeln('Fizz!')
      else if number mod 5 = 0 then
        writeln('Buzz!')
      else
        writeln(number)
  end;
end.

File: fizzbuzz.java
public class fizzbuzz {
  public static void main(String[] args) {
    final Integer finish = 20;
    int number = 0;

    while (number < finish) {
      number++;

      if (number % 3 == 0 && number % 5 == 0) {
        System.out.println("FizzBuzz!");
      } else if (number % 3 == 0) {
        System.out.println("Fizz!");
      } else if (number % 5 == 0) {
        System.out.println("Buzz!");
      } else {
        System.out.println(number);
      }
    }
  }
}
